<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LimeSDR UDP Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="fftLenSlider">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>22</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fftLenSpinBox">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>22</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>FFTN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="gainSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>73</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>73</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="gainSpinBox">
        <property name="maximum">
         <number>73</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>GAIN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="centerSlider">
        <property name="minimum">
         <number>30000</number>
        </property>
        <property name="maximum">
         <number>3800000</number>
        </property>
        <property name="singleStep">
         <number>10000</number>
        </property>
        <property name="pageStep">
         <number>10000</number>
        </property>
        <property name="value">
         <number>150000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="centerSpinBox">
        <property name="minimum">
         <number>30000</number>
        </property>
        <property name="maximum">
         <number>3800000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>CENTER</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSlider" name="spanSlider">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spanSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>SPAN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="reCalButton">
      <property name="text">
       <string>calibrate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="monitorButton">
      <property name="text">
       <string>monitor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="streamButton">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
